### DISCUSSION TITLE
Selecting code for copying ends up scrolling up to bottom in no time.

### DISCUSSION THREAD

**Discussion by imvickykumar999** [twbs/bootstrap#39637] [General] [answered]
https://github.com/twbs/bootstrap/assets/50515418/d9c72eba-21e2-4084-b156-d38c0f682d51

**Comment by julien-deramond** [comment, marked as answer]
Thanks for reporting this issue @imvickykumar999 
It's already been identified in our issues at https://github.com/twbs/bootstrap/issues/39605
